main.js

```diff
@@ -612,7 +612,15 @@ window.onload = async() => {
         });
     });
 
-    if(account !== ERR_ACCT) await walletClient.switchChain({ id: chain.id })
+    try {
+        // check connection to arb
+        await walletClient.switchChain({ id: chain.id })
+        await bank.read.par()
```

Comment:
For me on metamask and frame it reverts trying to switch chain if it isn't already in wallet. So the par read can be removed to speed up page load in normal case unless there's another reason for it